services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/Application.java

```diff
@@ -39,6 +39,7 @@
 @ComponentScan("org.eclipse.hono.service.auth")
 @ComponentScan("org.eclipse.hono.service.metric")
 @ComponentScan("org.eclipse.hono.deviceregistry")
+@ComponentScan("org.eclipse.hono.service.management.device")
```

Comment:
why do we need this additional annotation?

Reply:
My bad. We do not need this with the current solution.